mobile/src/global.css

```
@@ -149,5 +149,49 @@
 
       --color-placeholder: hsl(0 0% 100%);
     }
+
+    @variant atmosphere {
+      /*? All "primary" colors are used. */
```

Stylelint (17.14.1)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)